Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3/Utilities/Motor.mo

```diff
@@ -3,11 +3,12 @@ model Motor "Motor model including current controller of r3 motors"
   extends Electrical.Machines.Icons.Machine;
   parameter SI.Inertia J(min=0)=0.0013 "Moment of inertia of motor";
   parameter Real k=1.1616 "Gain of motor";
-  parameter Real w=4590 "Time constant of motor";
-  parameter Real D=0.6 "Damping constant of motor";
+  parameter Real w(unit="s-1")=4590 "Time constant of motor";
```

Comment:
```suggestion
  parameter SI.AngularFrequency w=4590 "Angular frequency of motor";
```

Time constants are usually given in seconds on motor data sheets. Hence I wonder if this rather is of quantity angular frequency.

Reply:
I didn't change the description in this PR.

It could be that someone gave the inverse time constant and forgot "inverse".

For some related cases the angular frequency is the inverse of the time constant; so that change is plausible (and w is often used for the angular frequency parameter) - but I don't know https://en.wikipedia.org/wiki/RC_time_constant#Cutoff_frequency

Reply:
Ah, well doing it